github/tests/Project.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+
+# ########################## Copyrights and license ############################
+#                                                                              #
+# Copyright 2018 bbi-yggy <yossarian@blackbirdinteractive.com>                 #
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.github.io/PyGithub/v1/index.html                             #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+# ##############################################################################
+
+import Framework
+import github
+
+class Project(Framework.TestCase):
+    def setUp(self):
+        # Force token authenticaton mode, since that's how the replay data was recorded.
+        self.tokenAuthMode = True

```

Comment:
I see what's you are doing here. Normally I just manually replace all the `token private_token_removed` with `Basic login_and_password_removed`. It's a bug obviously. I will think about how to improve it.

Reply:
Oh, does that work? Could the test data recorder just take care of that?

Reply:
It works for me. Well the test framework is quite old already, I'd rather not touch it in this PR. If needed, we can discuss it in a new thread dedicated to improving the tests framework. I've been exploring the possibility to use something vcr.py or betamax, but that seems a lot of work too.

Reply:
Aha, yes, search and replace for authentication mode works. I will update CONTRIBUTING.md